extensions/wikidata/src/org/openrefine/wikidata/schema/WikibaseSchema.java

```diff
@@ -57,38 +57,39 @@
 
     final static Logger logger = LoggerFactory.getLogger("RdfSchema");
 
-    @JsonProperty("itemDocuments")
-    protected List<WbItemDocumentExpr> itemDocumentExprs = new ArrayList<WbItemDocumentExpr>();
+    protected List<WbItemDocumentExpr> itemDocumentExprs = new ArrayList<>();
 
-    protected String baseIri = "http://www.wikidata.org/entity/";
+    protected String baseIri;
 
     /**
      * Constructor.
      */
     public WikibaseSchema() {
 
     }
-    
+
     /**
      * Constructor for deserialization via Jackson
      */
     @JsonCreator
-    public WikibaseSchema(@JsonProperty("itemDocuments") List<WbItemDocumentExpr> exprs) {
+    public WikibaseSchema(@JsonProperty("itemDocuments") List<WbItemDocumentExpr> exprs,
+                          @JsonProperty("entityPrefix") String baseIri) {
         this.itemDocumentExprs = exprs;
+        this.baseIri = baseIri;
     }
 
     /**
      * @return the site IRI of the Wikibase instance referenced by this schema
      */
-    @JsonIgnore
+    @JsonProperty("entityPrefix")
```

Comment:
Is it too late to sync up these names?

Reply:
It is worth making the code and the JSON format consistent but I would advocate to keep the `@JsonProperty` annotations anyway.